clang/lib/AST/Decl.cpp

```diff
@@ -5769,11 +5776,11 @@ void HLSLBufferDecl::addLayoutStruct(CXXRecordDecl *LS) {
   addDecl(LS);
 }
 
-void HLSLBufferDecl::addDefaultBufferDecl(Decl *D) {
+void HLSLBufferDecl::setDefaultBufferDecls(ArrayRef<Decl *> Decls) {
```

Comment:
should you be copying the ArrayRef instead of  in CreateDefaultCBuffer? It doesn't look like this function is used anywhere else, but could it be?

Reply:
Good idea!